src/sagas.ts

```diff
@@ -0,0 +1,44 @@
+import { setHeroesError, setCharacters, setIsLoading as setIsHeroesLoading, GET_HEROES } from './actions/search';
+import { Action } from 'redux';
+import { call, put, takeLatest } from 'redux-saga/effects';
+import { handleRequest } from './api/requestHandler';
+import { GET_COMICS, setComics, setComicsError, setIsLoading as setIsComicsLoading  } from './actions/comics';
+
+interface ITask {
+  payload: string;
+}
+
+interface TaskAction extends Action, ITask {
```

Comment:
for what?

Reply:
It's a workaround for typing issues. `takeLatest` throws errors without it